tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+from dotenv import load_dotenv
+
+
+# pylint: disable-next=unused-argument
+def pytest_configure(config):
+    """
+    Load environment variables before tests are run.
+    """
+    load_dotenv(".env.test")
```

Comment:
There is no need to load the `shared` dotenv?

Reply:
Good question! Shared is only relevant for the development environment imo. So no need to load it, but we can change it in the future if it's annoying.